scss/_side-navigation.scss

```diff
@@ -0,0 +1,188 @@
+  --#{$prefix}offcanvas-border-width: 0;
+
+  width: var(--#{$prefix}side-nav-width);
+  height: 100%;
+  overflow: hidden auto;
+
+  svg,
+  img {
+    width: var(--#{$prefix}side-nav-icon-size);
+    min-width: var(--#{$prefix}side-nav-icon-size);
+    height: var(--#{$prefix}side-nav-icon-size);
+    margin-right: var(--#{$prefix}side-nav-icon-margin-x);
+  }
+
+  .offcanvas-header,
+  .offcanvas-body > :not(.collapse-horizontal) {
+    width: 100%;
+    background: var(--#{$prefix}side-nav-bg);
+    border-right: var(--#{$prefix}side-nav-border-width) solid var(--#{$prefix}side-nav-border-color);
+  }
+
+  .offcanvas-body,
+  .offcanvas-body > :not(.collapse-horizontal) {
+    min-width: var(--#{$prefix}side-nav-width);
+    min-height: 100%;
+    background: var(--#{$prefix}side-nav-bg);
+  }
+
+  .offcanvas-body {
+    min-height: subtract(100%, var(--#{$prefix}side-nav-item-height));
+  }
+
+  .accordion {
+    --#{$prefix}accordion-border-width: 0;
+    --#{$prefix}accordion-body-padding-x: 0;
+    --#{$prefix}accordion-body-padding-y: 0;
+    --#{$prefix}accordion-btn-font-size: 1rem;
+
+    .accordion-body {
+      --#{$prefix}side-nav-item-padding-left: calc(var(--#{$prefix}side-nav-icon-size) + var(--#{$prefix}side-nav-icon-margin-x) * 2); // stylelint-disable-line function-disallowed-list
+
+      .accordion-body {
+        --#{$prefix}side-nav-item-padding-left: calc(var(--#{$prefix}side-nav-icon-size) * 2 + var(--#{$prefix}side-nav-icon-margin-x) * 3); // stylelint-disable-line function-disallowed-list
+      }
+    }
+  }
+
+  .tab-content { // to change after #1427 is merged
+    padding: 0;
+    border: 0;
+  }
+
+  .collapse-horizontal {
+    height: 0;
+    min-height: unset;
+    margin-left: var(--#{$prefix}side-nav-width);
+
+    > * {
+      width: calc(var(--#{$prefix}offcanvas-width) - var(--#{$prefix}side-nav-width)); // stylelint-disable-line function-disallowed-list
+      max-width: calc(100vw - var(--#{$prefix}side-nav-border-width) * 2 - var(--#{$prefix}side-nav-width)); // stylelint-disable-line function-disallowed-list
+    }
+
+    &:not(.show):not(.collapsing) + * .tab-content {
+      display: none;
+    }
+  }
+
+  .side-nav-item { // Here to be more specific than other classes
+    display: flex;
+    align-items: center;
+    width: 100%;
+    height: var(--#{$prefix}side-nav-item-height);
+    padding: 0 0 0 var(--#{$prefix}side-nav-item-padding-left);
+    margin: 0;
+    overflow: hidden;
+    color: var(--#{$prefix}side-nav-color);
+    text-decoration: none;
+    white-space: nowrap;
+    background-color: var(--#{$prefix}side-nav-bg);
+    border: 0;
+
+    &:hover {
+      background-color: var(--#{$prefix}side-nav-hover-bg);
+    }
+
+    &.active {
+      background: linear-gradient(to right, var(--#{$prefix}side-nav-active-border-color), var(--#{$prefix}side-nav-active-border-color) 4px, var(--#{$prefix}side-nav-active-bg) 4px, var(--#{$prefix}side-nav-active-bg));
```

Comment:
Grey text on a light grey background, I think we should check if the color constrat will be good enough.

Reply:
I think that this point relies on the user, am I right ?